ui/job-view/details/summary/ActionBar.jsx

```diff
@@ -308,6 +308,10 @@ class ActionBar extends React.PureComponent {
     try {
       const interactiveTask = getAction(results.actions, 'create-interactive');
 
+      if (user.email === '') {
+        await TaskclusterModel.getCredentials(currentRepo);
```

Comment:
Failure: `Taskcluster: can't access property "tc_root_url", currentRepo is undefined`

Reply:
same story, user creds, not tc creds